components/ServicesList.js

```
@@ -0,0 +1,28 @@
+  if (loading) return <p>Loading services…</p>
+  if (error) return <p>Error loading services.</p>
+
+  const items = data?.services?.nodes || []
+  if (items.length === 0) return <p>No services found.</p>
+
+  return (
+    <section className={styles.services} aria-labelledby="services-heading">
+      <h2 id="services-heading" className={styles.visuallyHidden}>Services</h2>
+      {items.map((s, i) => (
+        <article key={i} className={styles.serviceItem} aria-labelledby={`service-title-${i}`}>
+          <h3 id={`service-title-${i}`} dangerouslySetInnerHTML={{ __html: s.title }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)